webapp/src/main/webapp/src/app/sender/sync/sender-sync-message.component.html

```diff
@@ -27,8 +28,9 @@ <h5><span class="badge badge-dark" i18n="@@common-total-count">Total Count</span
 			<i class="bi-envelope-dash text-warning ui-cursor-pointer" [hidden]="m.status != 'NEW'" [ngbTooltip]="statusWaitingTemplate" [openDelay]="1000"></i>
 			<i class="bi-envelope-check-fill text-success ui-cursor-pointer" [hidden]="m.status != 'SENT'" [ngbTooltip]="statusSentTemplate" [openDelay]="1000"></i>
 		</td>
-		<td>{{m.dateCreated | date:'yyyy-MM-dd HH:mm:ss'}}</td>
-		<td>{{m.dateSent | date:'yyyy-MM-dd HH:mm:ss'}}</td>
+		<td container="body" ngbTooltip="{{m.eventDate | date:'EEEE, MMMM d, y, h:mm:ss'}}" [openDelay]="1000">{{m.eventDate | date:'MMM d, y'}}</td>
```

Comment:
date format should be `mediumDate` and tooltip date `EEEE, MMMM d, y, h:mm:ss a`

Reply:
That is the format of mediumDate 'MMM d, y'

Reply:
I know but lets use `dd` for date for all dates

Reply:
And for the tooltip to use dd and include AM/PM, this is more intuitive to users